experimental/record-tuple.md

```diff
@@ -0,0 +1,37 @@
+# [Record & Tuple][proposal-record-tuple]
+
+## Expressions
+
+### RecordExpression
+
+```js
+interface RecordExpression <: Expression {
+    type: "RecordExpression";
+    properties: [ RecordProperty | SpreadElement ];
+}
+```
+
+A record expression defines an immutable object, e.g. `#{a: 1}`.
+
+#### RecordProperty
+```js
+interface RecordProperty <: Node {
```

Comment:
We'll probably want to extract a `BaseProperty` mixin shared by both kinds of properties (object and record), but we can resolve this later if/when proposal is accepted, since it's a readability nit that doesn't change anything semantically.

Reply:
(Of course, another alternative here could be to keep the `"Property"` node type, but override `kind` to strictly `"init"` like we do on some other subtypes.)

Reply:
I like the idea of `BaseProperty` mixin! Note that we can also share it with `ClassProperty` or `ClassElement` defined in #180. Since it does not change semantic we can refactor in a separate PR when #180 is resolved.